File: Week 13 Multithread/Java/src/Result.java
public class Result {
    private final int taskId;
    private final String processedData;
    private final long processingTime;
    private final int workerId;

    public Result(int taskId, String processedData, long processingTime, int workerId) {
        this.taskId = taskId;
        this.processedData = processedData;
        this.processingTime = processingTime;
        this.workerId = workerId;
    }

    public int getTaskId() {
        return taskId;
    }

    public String getProcessedData() {
        return processedData;
    }

    public long getProcessingTime() {
        return processingTime;
    }

    public int getWorkerId() {
        return workerId;
    }

    @Override
    public String toString() {
        return "Result{taskId=" + taskId + 
               ", processedData='" + processedData + 
               "', processingTime=" + processingTime + 
               "ms, workerId=" + workerId + "}";
    }
}

File: Week 13 Go Multithread/Go/internal/model/result.go
package model

import "fmt"

// Result represents the output of a processed task
type Result struct {
	TaskID         int
	ProcessedData  string
	ProcessingTime int64
	WorkerID       int
}

// String returns a string representation of the result
func (r Result) String() string {
	return fmt.Sprintf("Result{taskId=%d, processedData='%s', processingTime=%dms, workerId=%d}",
		r.TaskID, r.ProcessedData, r.ProcessingTime, r.WorkerID)
}